### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by m-anwar97**
Made a replica of a youtube video page, with a video player, suggested videos, description section, and a comment section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+          <span class="like-history">
+            <i
+              class="material-icons"
+              style="cursor: pointer; margin-left: 10px; margin-right: 10px;"
+              >thumb_up</i
+            >
+            500
+            <i
+              class="material-icons"
+              style="cursor: pointer; margin-left: 10px; margin-right: 10px;"
```
Comment (expjazz) [inline index.html]: - [ ] Kindly remove all the inline selectors since they are not a good practice.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+                From Student to Web Developer:<br />An interview with Juan
+                Manuel..
+              </h5>
+
+              <i class="material-icons">more_vert</i>
+            </div>
+
+            <p style="margin: 5px 0 0 0; padding: 0;">Microverse</p>
+            <p style="margin: 0; padding: 0;">1.9K views . 7 Months ago</p>
+          </div>
```
Comment (expjazz) [inline index.html]: - [ ] I think the fonts-sizes could be closer to the original (channel name, number of views and comments). Make them smaller

| Yours | Original |
|:---:|:---:|
|image |image |.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,27 @@
-# Youtube-replica
-First microverse project
+> # Youtube-clone
+>
+> This is a youtube clone page made as the first project for the Microverse HTML&CSS3 Track. Is has the main section which has the main video player and a side section with suggested videos. Comments and descriptions are also displayed as they are on youtube.
+
+![screenshot](https://ibb.co/Yp0Gd05)
```
Comment (expjazz) [inline README.md]: - [ ] The screenshot is not appearing

**Review by expjazz — CHANGES_REQUESTED** [review]
## Welcome :+1: 

### Status: Changes Required :recycle: 

You did a very good job on this project :100: 

You made it responsive, the code is clean, well organized and I can see that you have a great attention to detail :clap: 

However, there are a few details (check the inline comments below) that will make the project even better. One thing that you did that you should not do again is to use styling inline in the HTML. You can check this article for a reference on why [https://speckyboy.com/good-bad-css-practices/](url)

Also, you can make the name of the pull request more meaningful, that represents more what you have achieved in this project.

#### As stated above, the project looks really good.

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:

**Comment by bertil291utn** [comment]
# **Status:** Approved ✅

## Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### And yes as my partner said, welcome to the program, you're going to love it

<h2 align="center">Ship it </h2>

<p align="center">image</p>

Please merge to master branch and move on with the next assignment.

You're doing  a great job keep going 💻 🚀 🔥

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

### OPTIONAL

-  Encourage the use of the [`<footer>`](https://www.w3schools.com/TagS/tag_footer.asp) tag for the footer

**Pull request merged**